### ISSUE TITLE
SQM Configurations Changes After OpenMPTCP Reboot

### ISSUE THREAD

**Issue by chubx** [Ysurac/openmptcprouter#3467]
## Expected Behavior

My working configurations should survive reboot.

## Current Behavior

So, I am bonding 1 Fibre connection with Starlink. SQM has been tested to work.

However, when I reboot the router and check on the SQM page again, the configurations has been changed.

I made sure that everything was saved/applied before rebooting, and tested working.

## Specifications

  - OpenMPTCProuter version: ``v0.60-6.1 r0+24843-acf40c022e
  - OpenMPTCProuter VPS version:  `OpenMPTCProuter VPS 0.1030`
  - OpenMPTCProuter VPS provider: Amazon AWS EC2
  - OpenMPTCProuter platform x86_64

**Comment by Ysurac** [comment]
What was changed exactly ?

**Comment by chubx** [comment]
> What was changed exactly ?

I was testing and experimenting, so I deleted the interfaces from the SQM page.

I then re-created them, and configured the Download/Upload speeds. then clicked on Save/Apply.

I do the tests and it works, SQM works according to my needs.

I then reboot the router, then visit the SQM page again, but configurations had change, as to Download and Upload setting, not something that I've configured.
